GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigAdvancedSettingsPage.Designer.cs

```diff
@@ -31,47 +31,60 @@ private void InitializeComponent()
             // 
             this.checkBoxRebaseAutostash.AutoSize = true;
-            this.checkBoxRebaseAutostash.Location = new System.Drawing.Point(19, 80);
+            this.checkBoxRebaseAutostash.Location = new System.Drawing.Point(19, 82);
             this.checkBoxRebaseAutostash.Name = "checkBoxRebaseAutostash";
-            this.checkBoxRebaseAutostash.Size = new System.Drawing.Size(228, 17);
+            this.checkBoxRebaseAutostash.Size = new System.Drawing.Size(363, 19);
             this.checkBoxRebaseAutostash.TabIndex = 3;
-            this.checkBoxRebaseAutostash.Text = "Automatically stash before doing a rebase";
+            this.checkBoxRebaseAutostash.Text = "Automatically stash before doing a rebase i.e. `rebase.autoStash`";
             this.checkBoxRebaseAutostash.UseVisualStyleBackColor = true;
             // 
             // checkBoxFetchPrune
             // 
             this.checkBoxFetchPrune.AutoSize = true;
-            this.checkBoxFetchPrune.Location = new System.Drawing.Point(19, 47);
+            this.checkBoxFetchPrune.Location = new System.Drawing.Point(19, 48);
             this.checkBoxFetchPrune.Name = "checkBoxFetchPrune";
-            this.checkBoxFetchPrune.Size = new System.Drawing.Size(199, 17);
+            this.checkBoxFetchPrune.Size = new System.Drawing.Size(302, 19);
             this.checkBoxFetchPrune.TabIndex = 2;
-            this.checkBoxFetchPrune.Text = "Prune remote branches during fetch";
+            this.checkBoxFetchPrune.Text = "Prune remote branches during fetch i.e `fetch.prune`";
             this.checkBoxFetchPrune.UseVisualStyleBackColor = true;
             // 
             // checkBoxPullRebase
             // 
             this.checkBoxPullRebase.AutoSize = true;
             this.checkBoxPullRebase.Location = new System.Drawing.Point(19, 14);
             this.checkBoxPullRebase.Name = "checkBoxPullRebase";
-            this.checkBoxPullRebase.Size = new System.Drawing.Size(276, 17);
+            this.checkBoxPullRebase.Size = new System.Drawing.Size(387, 19);
             this.checkBoxPullRebase.TabIndex = 1;
-            this.checkBoxPullRebase.Text = "Rebase local branch when pulling (instead of merge)";
+            this.checkBoxPullRebase.Text = "Rebase local branch when pulling (instead of merge) i.e. `pull.rebase`";
             this.checkBoxPullRebase.UseVisualStyleBackColor = true;
             // 
+            // checkBoxRebaseAutosquash
+            // 
+            this.checkBoxRebaseAutosquash.AutoSize = true;
+            this.checkBoxRebaseAutosquash.Location = new System.Drawing.Point(19, 116);
+            this.checkBoxRebaseAutosquash.Name = "checkBoxRebaseAutosquash";
+            this.checkBoxRebaseAutosquash.Size = new System.Drawing.Size(492, 19);
+            this.checkBoxRebaseAutosquash.TabIndex = 3;
+            this.checkBoxRebaseAutosquash.Text = "Automatically squash commits when doing an interactive rebase i.e. `rebase.autosq" +
```

Comment:
keep on one line (or explicitly break earlier so it can be searched in VS)

Reply:
Not possible with the last refactoring because the config key is no more part of the control text (it is added at load time) and I find it better because translation doesn't change and translator doesn't have to take care of that.